docs: :memo: add organizational GitHub profile page (#307)
seedcase-project/.github: profile/README.md

<p align="center">
  <img src="https://raw.githubusercontent.com/seedcase-project/seedcase-theme/main/_extensions/seedcase-theme/logos/seedcase-logo.svg" alt="Seedcase Project logo." width="150"/>
</p>

<p align="center">
  <img src="https://raw.githubusercontent.com/seedcase-project/seedcase-theme/main/_extensions/seedcase-theme/logos/seedcase-font.svg" alt="Seedcase text as an image." width="150"/>
</p>

# A framework for building FAIR research data

Our primary goal is to build open software solutions that help:

-   Research groups and institutions create and manage high-quality and
    FAIR (Findable, Accessible, Interoperable, and Reusable) research
    data.
-   Individual researchers to find and (re-)use data more easily for
    their own research.
-   Reduce the time from data generation to scientific dissemination by
    focusing on streamlining and automating the data ingestion,
    processing, storage, and management processes.

Check our [website](https://seedcase-project.org/) for more details, such as

> our roadmap, products, team, and other websites, as well as our collaborators.

Reviewer: ```suggestion
our roadmap, products, team, links to our other websites, and our collaborators.
```
